bitrise.yml

```diff
@@ -49,6 +49,9 @@ workflows:
     - cocoapods-install@2.2:
         inputs:
         - podfile_path: $FRAMES_PODFILE
+    - swiftlint-extended@1:
```

Comment:
@ehab-al-cko as discussed we can remove this step for now. Since Raywanderlich only defines one error for hash rules. We need to agree what is error and what is warning in next iOS dev meeting so that this step will have more value / meaning. Otherwise currently this will mostly pass 95 % of the time as everything is defined as a warning in yml file and would be a 4 second overkill on each build.